CONTRIBUTING.md

```diff
@@ -62,6 +62,12 @@ see [ABI Policy Guide](sycl/doc/ABIPolicyGuide.md) for more information.
 - For any DPC++-related commit, the `[SYCL]` tag should be present in the
   commit message title. To a reasonable extent, additional tags can be used
   to signify the component changed, e.g.: `[PI]`, `[CUDA]`, `[Doc]`.
+- For product changes which require modification of E2E tests
+  (see [Test DPC++ toolchain](sycl/doc/GetStartedGuide.md#test-dpc-toolchain))
+  the commit message should contain link to corresponding test PR, e.g.: E2E
```

Comment:
Could you please clarify if the intent here is that a link should be in final commit message or in the first PR message or both?

Reply:
Thinking more about that... Should we also require that PR to have clear message which will be used as final merge commit message when many commits squashed?
If we do this link to test should be in PR.

Reply:
done